src/main/java/org/eln2/mc/Eln2.kt

```diff
@@ -28,7 +31,9 @@ object Eln2 {
         ItemRegistry.setup(MOD_BUS)
         ContainerRegistry.setup(MOD_BUS)
 
-        MOD_BUS.register(ContainerRegistry) //client-side setup
+        if (Dist.CLIENT == FMLEnvironment.dist) {
```

Comment:
Does this work standalone?

Reply:
Yes, it should provide the physical side, determined at startup.

If we have issues, moving to a DistExecutor is simple enough - and would probably be good to do in the future anyway.